app/src/main/java/swati4star/createpdf/activity/CropImageActivity.java

```diff
@@ -94,13 +118,38 @@ public void cropButtonClicked() {
         mCropImageView.saveCroppedImageAsync(Uri.fromFile(file));
     }
 
+
+    public void cropModeButtonClicked() {
+        mUriStored = mCropImageView.getImageUri();
+        Bitmap image = mCropImageView.getCroppedImage();
+        setContentView(mCropView);
+        Button okButton = findViewById(R.id.CropOKButton);
+        okButton.setOnClickListener(view -> cropOKButtonClicked());
+        Button cancelButton = findViewById(R.id.CropCancelButton);
+        cancelButton.setOnClickListener(view -> cropCancelButtonClicked());
+        me.pqpo.smartcropperlib.view.CropImageView ivCrop = findViewById(R.id.iv_crop);
+        ivCrop.setImageToCrop(image);
+
+    }
+
+    public void cropOKButtonClicked() {
+        me.pqpo.smartcropperlib.view.CropImageView ivCrop = findViewById(R.id.iv_crop);
+        Bitmap crop = ivCrop.crop();
+        mCropImageView.setImageBitmap(crop);
+        setContentView(mMainView);
+    }
+
+    public void cropCancelButtonClicked() {
+        setContentView(mMainView);
+    }
+
     public void rotateButtonClicked() {
         mCurrentImageEdited = true;
         mCropImageView.rotateImage(90);
     }
 
     public void nextImageClicked() {
-        if ( mImages.size() == 0)
+        if (mImages.size() == 0)
```

Comment:
I know these unnecessary changes are not any big deal for now, but they can confuse if anyone watch these commit later. These all beautification process and other thing deserve separate commit.